cypress/integration/1216-menu/user-sees-menu/user-sees-menu.js

```diff
@@ -0,0 +1,98 @@
+import { Given, When, Then, And, Before } from 'cypress-cucumber-preprocessor/steps';
+import {
+  USER_CURATOR_WITH_AUTHOR,
+  USER_NO_ROLE,
+  USER_WITH_AUTHOR,
+  ADMIN_USER,
+  USER_INST_ADMIN_WITH_AUTHOR,
+} from '../../../support/constants';
+import {
+  ADMIN_MENU,
+  CREATOR_MENU,
+  CURATOR_MENU,
+  INST_ADMIN_MENU,
+  MAIN_BUTTONS,
+  USER_MENU,
+} from '../../../support/data_testid_constants';
+import { v4 as uuidV4 } from 'uuid';
+
+// Feature: User sees menu
+
+// Common steps
+
+Before({ tags: '@345' }, () => {
+  cy.wrap(USER_NO_ROLE).as('USER');
+  cy.wrap(USER_MENU).as('MENU');
+});
+
+Before({ tags: '@346' }, () => {
+  cy.wrap(USER_WITH_AUTHOR).as('USER');
+  cy.wrap(CREATOR_MENU).as('MENU');
+});
+
+Before({ tags: '@347' }, () => {
+  cy.wrap(USER_CURATOR_WITH_AUTHOR).as('USER');
+  cy.wrap(CURATOR_MENU).as('MENU');
+});
+
+Before({ tags: '@348' }, () => {
+  cy.wrap(USER_INST_ADMIN_WITH_AUTHOR).as('USER');
+  cy.wrap(INST_ADMIN_MENU).as('MENU');
+});
+
+Before({ tags: '@350' }, () => {
+  cy.wrap(ADMIN_USER).as('USER');
+  cy.wrap(ADMIN_MENU).as('MENU');
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@USER').then((user) => {
+    cy.login(user);
+  });
+});
+When('they look at any page in NVA', () => {
+  cy.visit(`/${uuidV4()}`);

```

Comment:
veldig bokstavelig :p i praksis blir dette å se not-found hver gang da så kan like godt bruke bare `/`, men går for det samme